src/pages/components/CertificateInfo.qml

```diff
@@ -103,146 +79,103 @@ SilicaFlickable {
+        MouseArea {
+            id: permissionArea
+            width: parent.width
+            height: permissionColumn.height
+            onClicked: openSiteSettings()
+
+            Column {
+                id: permissionColumn
+                width: parent.width
+                spacing: Theme.paddingMedium
+                anchors.horizontalCenter: parent.horizontalCenter
+                visible: permissionIndicationRepeater.count > 0
+
+                Label {
+                    width: parent.width - 2 * Theme.horizontalPageMargin
+                    x: Theme.horizontalPageMargin
+                    horizontalAlignment: Text.AlignHCenter
+                    wrapMode: Text.Wrap
+                    color:  Theme.highlightColor
+                    //% "Current permissions"
+                    text: qsTrId("sailfish_browser-sh-current-permissions")
+                }
 
-        CertificateLabel {
-            //: Label prefixing the status of whether tracking tech is being used on a page
-            //% "Tracking"
-            key: qsTrId("sailfish_browser-la-cert_tracking")
-            value: {
-                if (security) {
-                    if (security.loadedTrackingContent) {
-                        //% "The page includes items used to track your behaviour"
-                        return qsTrId("sailfish_browser-la-cert_tracking_content_loaded")
-                    } else if (security.blockedTrackingContent) {
-                        //% "Items used to track your behaviour have been blocked"
-                        return qsTrId("sailfish_browser-la-cert_tracking_content_blocked")
+                Grid {
+                    rows: Math.ceil(permissionIndicationModel.count / 5)
+                    columns: Math.min(permissionIndicationModel.count, 5)
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    spacing: Theme.paddingLarge
+                    Repeater {
+                        id: permissionIndicationRepeater
+                        model: permissionIndicationModel
+                        delegate: Icon {
+                            source: {
+                                var blocked = (model.capability === PermissionManager.Deny)
+                                if (model.type === "geolocation") {
+                                    return "image://theme/icon-m-browser-location" + (blocked ? "-template" : "")
+                                } else if (model.type === "cookie") {
+                                    return "image://theme/icon-m-browser-cookies" + (blocked ? "-template" : "")
+                                } else if (model.type === "popup") {
+                                    return "image://theme/icon-m-browser-popup" + (blocked ? "-template" : "")
+                                }
+                                highlighted: permissionArea.pressed
+                            }
+                            color: Theme.highlightColor
```

Comment:
Since these are now pressable, I think the default colour should be used instead (in which case, this line can be removed).